src/tools/miri/src/lib.rs

```diff
@@ -16,7 +16,6 @@
 #![feature(unqualified_local_imports)]
 #![feature(derive_coerce_pointee)]
 #![feature(arbitrary_self_types)]
-#![feature(unsigned_is_multiple_of)]
```

Comment:
I'm never sure but I guess CI will tell us: should miri be updated, or will that happen when they sync from their separate repo?

Reply:
You'll have to update Miri here to make CI pass. :)  A bot will take care of syncing that to the Miri repo when this PR lands.